script/install.sh

```diff
@@ -119,10 +121,39 @@ fetch() {
 
 # ------------------------------------------------------------------------------
 
+# Returns false if the script is invoked from a Windows command prompt.
+posix_terminal() {
+    # If run from a POSIX terminal (including under MSYS2 or Cygwin) the TERM
+    # variable is defined to something like "xterm". If run from a Windows
+    # command prompt through bash.exe from an MSYS installation, TERM keeps
+    # its default value, which is "cygwin".
+    if [[ $TERM == "cygwin" ]]; then
+        return 1;   # false
+    else
+        return 0;   # true
```

Comment:
This seems sketchy and will confuse future readers.

Reply:
Would this be better?
```
    if [[ $TERM == "cygwin" ]]; then
        return `false`;
    else
        return `true`;
    fi
```

Reply:
I'll change it into
```
    if [[ $TERM == "cygwin" ]]; then
        return $(false);
    else
        return $(true);
    fi
```